s3prl/run_downstream.py

```diff
@@ -134,7 +138,7 @@ def update_args(old, new, preserve_list=None):
         if args.upstream_model_config is not None and os.path.isfile(args.upstream_model_config):
             backup_files.append(args.upstream_model_config)
 
-    if args.override:
+    if args.override and args.override != "defaults":
```

Comment:
@leo19941227 here is where I am checking against the Docker environment variable, so that by default we don't trigger the override of `config.yaml`

Reply:
Thanks! Perhaps we can use `args.override.lower() != "none"`?

Reply:
Good idea!